### PR TITLE
Feature/8191 profile edit delete buttons

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
This PR adds the additional action sheet to navigate to the profile-info or profile-edit screen.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8191

## Screenshots

Simulator Screen Shot - iPhone 12 - 2021-06-30 at 14 09 54
Simulator Screen Shot - iPhone 12 - 2021-06-30 at 14 09 57

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -847,6 +847,12 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 				)
 
 			},
+			didTapInfoProfile: {
+				// Todo: Navigate to profile info
```
Comment (30mar) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:851]: Are these going to be done in another ticket ?
```suggestion
				// Todo: Navigate to profile info
```

**Review by 30mar — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:851]: Yes

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by 30mar — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift [hunk 2]
```diff
@@ -11,11 +11,15 @@ class AntigenTestProfileViewController: UIViewController, UITableViewDataSource,
 	init(
 		store: AntigenTestProfileStoring,
 		didTapContinue: @escaping (@escaping (Bool) -> Void) -> Void,
+		didTapInfoProfile: @escaping () -> Void,
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift:14]:
```suggestion
		didTapProfileInfo: @escaping () -> Void,
```

Just a tiny improvement

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift [hunk 3]
```diff
@@ -70,26 +74,19 @@ class AntigenTestProfileViewController: UIViewController, UITableViewDataSource,
...
-					self?.didTapDeleteProfile()
-				}
-			)
-			alert.addAction(deleteAction)
-			
-			let cancelAction = UIAlertAction(title: AppStrings.Common.alertActionCancel, style: .cancel)
-			alert.addAction(cancelAction)
-			
-			present(alert, animated: true, completion: nil)
+			let ac = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift:77]:
```suggestion
			let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
```

**Review by freshking — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift (hunk 3 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift:77]: Did this change on a following PR.

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift (hunk 2 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift:14]: Will change this on a following PR. So this one can be merge immediately.

**Pull request merged**